src/components/vive-controls.js

```diff
@@ -1,25 +1,29 @@
 var registerComponent = require('../core/component').registerComponent;
 var bind = require('../utils/bind');
+var trackedControlsUtils = require('../utils/tracked-controls');
 
 var VIVE_CONTROLLER_MODEL_OBJ_URL = 'https://cdn.aframe.io/controllers/vive/vr_controller_vive.obj';
 var VIVE_CONTROLLER_MODEL_OBJ_MTL = 'https://cdn.aframe.io/controllers/vive/vr_controller_vive.mtl';
 
 /**
  * Vive Controls Component
- * Interfaces with vive controls and maps Gamepad events to
- * vive controller buttons: trackpad, trigger, grip, menu and system
- * It loads a vive controller model and highlights the pressed buttons
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
  */
 module.exports.Component = registerComponent('vive-controls', {
   dependencies: ['tracked-controls'],
 
   schema: {
     hand: {default: 'left'},
-    buttonColor: {default: '#FAFAFA'},  // Off-white.
+    buttonColor: { default: '#FAFAFA' },  // Off-white.
     buttonHighlightColor: {default: '#22D1EE'},  // Light blue.
-    model: {default: true}
+    model: {default: true},
+    rotationOffset: {default: 0} // use -999 as sentinel value to auto-determine based on hand
   },
 
+  idPrefix: 'OpenVR Gamepad',
```

Comment:
This should be a const `GAMEPAD_ID` the same as https://github.com/aframevr/aframe/pull/2132/files#diff-8eb3323319bd9dc995e7909096d022a6R5